docs/source/conf.py

```diff
@@ -63,16 +63,19 @@
 html_theme_options = {
     "collapse_navigation": True,
     "navigation_depth": 4,
+    "logo_only": True,
```

Comment:
@parano should we change docs home button to BentoML's logo?

Reply:
great idea!